artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/check/QueueCheck.java

```diff
@@ -179,7 +179,7 @@ private void checkQueueConsume(final CheckContext context) throws Exception {
          connection.start();
 
          if (consume == -1) {
-            queueConsumer.receiveNoWait();
+            System.out.println("REceived " + queueConsumer.receiveNoWait());
```

Comment:
Typo.

Reply:
it is a debug leftover.. thanks.. removing it.